Zotero/Controllers/Attachment Downloader/AttachmentDownloader.swift

```diff
@@ -796,11 +796,27 @@ extension AttachmentDownloader: URLSessionDownloadDelegate {
             return
         }
 
+        var error: Swift.Error?
+        do {
```

Comment:
No need to have this in `do catch` block, just have something like
```
let error = checkFileResponse(for: Files.file(from: location), fileStorage: fileStorage, downloadTask: downloadTask)

if let data = activeDownload.logData {
    logResponse(for: data, task: downloadTask, error: error)
}
...
if let error {
    ...
}
```

Reply:
Correct, I just merged this, I'll just push another commit to improve this.

Reply:
Check https://github.com/zotero/zotero-ios/commit/919a1d40e913e3b4182d0bc76cc0bbf4bab870f6 (commited directly)